### PR TITLE
Add: Added sample ruby code for keyboard->key_down

### PR DESCRIPTION
**Pull request by Harsha509**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/webdriver/keyboard.en.md [hunk 1]
```diff
@@ -1,193 +1,227 @@
...
+  {{< / code-panel >}}
+  {{< code-panel language="ruby" >}}
+require 'selenium-webdriver'
+driver = Selenium::WebDriver.for :chrome
+begin
+  # Navigate to URL
+  driver.get 'https://google.com'
+
+  # Enter text "cheese" and perform keyboard action "Enter"
+  driver.find_element(name: 'q').send_keys 'q', :return
```
Comment (diemol) [inline docs_source_files/content/webdriver/keyboard.en.md]: The code examples are not the same as the other ones, they are not sending `cheese`... Please double check when copying & pasting.
(Same applies to the Kotlin one)

Comment (Harsha509) [reply docs_source_files/content/webdriver/keyboard.en.md]: Sure @diemol , sorry for that. I will update the comments

Comment (diemol) [reply docs_source_files/content/webdriver/keyboard.en.md]: It is not only the comments, the actual code is not doing the same as the other examples.

Comment (Harsha509) [reply docs_source_files/content/webdriver/keyboard.en.md]: hi @diemol , the code sample is correct. only the comment is wrong. In all code samples it enters q for search and later it selects the page.

Comment (Harsha509) [reply docs_source_files/content/webdriver/keyboard.en.md]: and kotlin code just looks fine, i have tested multiple times. Can you please review again, i took hard time to test and replace the code didn't copy pasted :(

Comment (diemol) [reply docs_source_files/content/webdriver/keyboard.en.md]: Apologies, it just seems that overall the examples are sending different texts. I will merge and then fix it. 
Thanks!

Comment (Harsha509) [reply docs_source_files/content/webdriver/keyboard.en.md]: Sure, i will check the keyboard section again once site is updated. Sorry if i have done any mistake from my end too.

Thanks a lot @diemol.

**Pull request merged**